plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/Rules/ContextPluginsRuleProcessor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Automattic\WooCommerce\Internal\Admin\RemoteFreeExtensions\Rules;
+
+use Automattic\WooCommerce\Admin\RemoteSpecs\RuleProcessors\ComparisonOperation;
+use Automattic\WooCommerce\Admin\RemoteSpecs\RuleProcessors\RuleProcessorInterface;
+
+/**
+ * Rule processor for context_plugins rules.
+ *
+ * Processes the following rule:
+ * {
+ *     "type": "context_plugins",
+ *     "name": "name of a property in the plugin object",
+ *     "value": "value to match",
+ *     "operation": "operation"
+ * }
+ */
+class ContextPluginsRuleProcessor implements RuleProcessorInterface {
+
+	/**
+	 * The list of plugins.
+	 *
+	 * @var array a list of plugins.
+	 */
+	private array $plugins;
+
+	/**
+	 * Constructor.
+	 * @param array $plugins a list of plugins.
```

Comment:
Is this actually a list of objects? I first interpreted it as strings, e.g: `[  'woocommerce', 'jetpack'  ]`

Reply:
@adrianduffell Added a comment in [Add details for plugin obj](https://github.com/woocommerce/woocommerce/pull/56233/commits/bc78768c1b57ad9ccccebe5cd2754840eedf93ac)

Reply:
Nice, the new documentation helps a lot!